src/benchmarklib/tpcc/tpcc_table_generator.hpp

```diff
@@ -56,13 +56,13 @@ class TpccTableGenerator : public opossum::AbstractBenchmarkTableGenerator {
 
   std::shared_ptr<Table> generate_table(const std::string& table_name);
 
-  void encode_table(const std::string& table_name, std::shared_ptr<Table> table);
-
   const size_t _warehouse_size;
   const time_t _current_date = std::time(0);
   const std::optional<EncodingConfig> _encoding_config;
 
  protected:
+  void encode_table(const std::string& table_name, std::shared_ptr<Table> table);
```

Comment:
Uuuuh... with #945, this will be an error. ;)

`_encode_table` and `_generate_inner_order_line_column`.